daemon/create.go

```diff
@@ -16,6 +17,13 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 		return "", warnings, err
 	}
 
+	// The check for a valid workdir path is made on the server rather than in the
+	// client. This is because we don't know the type of path (Linux or Windows)
+	// to validate on the client.
+	if config.WorkingDir != "" && !filepath.IsAbs(config.WorkingDir) {
```

Comment:
I think this should be checked at some place like `runconfig.Parse` because nothing else about the container gets checked here and all other checks are apparently done somewhere else.